stellar-dotnet-sdk/responses/results/CreateClaimableBalanceSuccess.cs

```diff
@@ -9,7 +9,7 @@ public class CreateClaimableBalanceSuccess : CreateClaimableBalanceResult
 {
     public CreateClaimableBalanceSuccess(byte[] balanceId)
     {
-        BalanceId = Convert.ToBase64String(balanceId);
+        BalanceId = Convert.ToHexString(balanceId);
```

Comment:
Did you test this really well? and is this a breaking change?

Reply:
I did.
It's not a breaking change since it was only added in this PR.